src/Corrade/Utility/Sha1.h

```diff
@@ -51,16 +52,26 @@ class CORRADE_UTILITY_EXPORT Sha1: public AbstractHash<20> {
         explicit Sha1();
 
         /** @brief Add data for digesting */
+        Sha1& operator<<(Containers::ArrayView<const char> data);
+
+        /** @overload */
         Sha1& operator<<(const std::string& data);
 
+        /** @overload */
+        template<std::size_t size>
+        Sha1& operator<<(const char(&data)[size]) {
+            return *this << Containers::arrayView(data, size);
```

Comment:
What if the const char isn't actually a string, but e.g. `char a[2]{1, 2}`? Imo, this should probably also include the delimiter (or maybe not exist at all?).

Reply:
> not exist

but without this, it caused ambiguous overloads, no? Ideally it would be without this overload, yes. Any existing code that fails if you remove this one?

Reply:
In this case, probably the std::string override should also include the delimiter. That would diverge from what python doesn, though, I think.